src/devices/vdml_vision.c

```diff
@@ -64,7 +63,7 @@ vision_object_s_t vision_get_by_size(uint8_t port, const uint32_t size_id) {
 	}
 	err = vexDeviceVisionObjectGet(device->device_info, size_id, (V5_DeviceVisionObject*)&rtn);
```

Comment:
need to declare `err` here if you're going to remove it above. currently doesn't compile